definitions/variable/technology/electricity-operation.yaml

```diff
@@ -104,23 +110,47 @@
     description: Exported electricity from a given region
-    unit: MWh
+    unit: [MWh, GWh]
 - Unserved Demand|Electricity:
     description: Not served demand
-    unit: MWh
+    unit: [MWh, GWh]
+- Flow|Electricity:
+    description: balance of energy exchanges for a given line, described as region1>region2
+      if the variable is positive it means that more energy has been sent from region 1 to region 2 than from region2 to region1
+    unit: [MWh, GWh]
 - Injected Energy|Electricity|{Electricity Storage Type}:
-    description: Amount of energy charged to the storage during the period
+    description: Amount of energy charged from the grid to the storage during the period
     notes: this represents the total energy which is charged into the storage during the period, the energy which is discharged during 
      that same period is not substracted from this total. This means that the Net Charged Energy can be computed as Charged Energy - Discharged Energy
      if Charged Energy > Discharged Energy, 0 if not
-    unit: GWh
+    unit: [MWh, GWh]
 - Discharged Energy|Electricity|{Electricity Storage Type}:
-    description: Amount of energy charged to the storage during the period
-    notes: this represents the total energy which is discharged from the storage during the period, the energy which is charged into the storage during 
+    description: Amount of energy discharged from the storage to the grid during the period
+    notes: this represents the total energy which is discharged from the storage to the grid during the period, the energy which is charged into the storage during 
      that same period is not substracted from this total. This means that the Net Discharged Energy can be computed as Discharged Energy - Charged Energy,
      if Discharged Energy > Charged Energy, 0 if not
+    unit: [MWh, GWh]
+- Injected Energy|Electricity|Load Shifting|{Residential Load Shifting Asset}:
+    description: Surplus of energy consumed by the {Residential Load Shifting Asset} during the period,
+        compared to nominal consumption profile. This means that the <Residential Load Shifting Asset> is advancing some consumption
+        i.e. consuming earlier than in the nominal profile, or that the consumption had been lowered in an earlier period and is
+        consumed later (delayed)
+    notes: this is equivalent to modelling the <Residential Load Shifting Asset> as a virtual storage 
+      this represents the total energy which is charged into the storage during the period, the energy which is discharged during 
+     that same period is not substracted from this total. This means that the Net Charged Energy can be computed as Charged Energy - Discharged Energy
+     if Charged Energy > Discharged Energy, 0 if not
+    unit: [MWh, GWh]
+- Discharged Energy|Electricity|Load Shifting|{Residential Load Shifting Asset}:
+    description: Amount of energy not consumed by the {Residential Load Shifting Asset} during the period,
```

Comment:
I think this variable is contrary to the "Injected...", but I think it is not totally intuitive. The discharged energy is implicit or a consequence of reducing consumption. Perhaps renaming both variables as increasing/decreasing load could be more intuitive. What is your view?

Reply:
@erikfilias I did not want to introduce new variable names and stick to those already used for the short term storages.... as it may help treatments of results....  
So What we could do is both keep these variables as they are (they are intuitive for people for whom it is intuitive to model load shifting as short term storages), and also add 2 new variables which could be 
- Increased consumed energy|Electricity|Load Shifting|{Residential Load Shifting Asset}
- Decreased consumed energy|Electricity|Load Shifting|{Residential Load Shifting Asset}
What do you think?
@danielhuppmann any opinion?

Reply:
I advise against having two variables that are basically synonymous, this will just create confusion

Reply:
Thanks Daniel, I agree also ; I suggest to keep this pr as it is 
Erik has approved